src/iracingdataapi/client.py

```diff
@@ -821,11 +861,19 @@ def member_info(self):
         """
         return self._get_resource("/data/member/info")
 
+    def member_participation_credits(self):
```

Comment:
I've avoided adding any that are specific only to the authenticated member, and I'm still erring on the side of this probably being unnecessary for this wrapper, but I'm up for being convinced.